ynh_check_starting/ynh_check_starting

```diff
@@ -2,46 +2,64 @@
 
 # Start or restart a service and follow its booting
 #
-# usage: ynh_check_starting "Line to match" [Log file] [Timeout]
+# usage: ynh_check_starting "Line to match" [service name] [Log file] [Timeout]
 #
 # | arg: Line to match - The line to find in the log to attest the service have finished to boot.
 # | arg: Log file - The log file to watch
 # /var/log/$app/$app.log will be used if no other log is defined.
 # | arg: Timeout - The maximum time to wait before ending the watching. Defaut 300 seconds.
 ynh_check_starting () {
 	local line_to_match="$1"
-	local app_log="${2:-/var/log/$app/$app.log}"
-	local timeout=${3:-300}
+	local service_name="${2:-$app}"
```

Comment:
Because in most cases the service name is going to be $app, and because we use positionnal arguments (and not getopts), it would be really better to have 'service_name' as the third argument, and not the first one.
Here, if we want to use app_log or timeout, we have to specify service_name.